roles/traefik/templates/traefik_dynamic.yaml

```diff
@@ -87,7 +87,7 @@ http:
       service: "dummy"
 
     minio:
-      rule: "Host(`{{ traefik_domain | default(hostvars[groups['hpc_master'][0]].ansible_ssh_host) }}`) && PathPrefix(`/minio`)"
+      rule: "Host(`{{ traefik_domain | default(hostvars[groups['hpc_master'][0]].ansible_ssh_host) }}`)"
```

Comment:
Keeping the old `/mino` would require some changes on how conda-store prepares the URLs which would not be feasible at the moment

Reply:
This looks like to be part of the S3 compatibility scheme, though couldn't find a ref. link besides this https://github.com/minio/minio-py/issues/1140#issuecomment-892683029 comment.